circe-akka-serializer/src/main/scala/org/virtuslab/ash/AkkaCodecs.scala

```diff
@@ -0,0 +1,32 @@
+package org.virtuslab.ash
+
+import akka.actor
+import akka.actor.typed.{ActorRef, ActorRefResolver, ActorSystem}
+import akka.serialization.Serialization
+import akka.stream.{SinkRef, SourceRef, StreamRefResolver}
+import io.circe.{Codec, Decoder, Encoder}
+
+trait AkkaCodecs {
+  private def serializationSystem: actor.ActorSystem = Serialization.getCurrentTransportInformation().system
+
+  implicit def actorRefCodec[T](implicit system: actor.ActorSystem = serializationSystem): Codec[ActorRef[T]] = {
```

Comment:
I'm always a bit wary of default params, let alone implicit default params... what would happen from the library's user perspective if the param was un-defaultized here?

Reply:
E.g. would the PR \#1122 in Hydra still compile?

Reply:
Right now it would only require making one argument of `CirceAkkaSerializer` implicit. But if code structure in Hydra evolves in a way it seems to be heading, then it will require this code again.